src/results/proforma.jl

```diff
@@ -284,10 +347,20 @@ function update_metrics(m::Metrics, p::REoptInputs, tech::AbstractTech, tech_nam
     total_kw = results[tech_name]["size_kw"]
     existing_kw = :existing_kw in fieldnames(typeof(tech)) ? tech.existing_kw : 0
     new_kw = total_kw - existing_kw
-    capital_cost = new_kw * tech.installed_cost_per_kw
+    if tech_name == "CHP"
+        capital_cost = get_chp_initial_capex(p, results["CHP"]["size_kw"])
+    else
+        capital_cost = new_kw * tech.installed_cost_per_kw
+    end
 
-    # owner is responsible for both new and existing PV maintenance in optimal case
-    if third_party
+    # owner is responsible for both new and existing PV (or Generator) maintenance in optimal case
```

Comment:
Given the if statement below, shouldn't this instead be "owner is responsible for only new_kw PV (or Generator) maintenance in optimal case." ?

Reply:
Yes, and since I am going to update Generator to do that too, I'm just going to say "new technologies" (even though Generator will not use this function)